### PR TITLE
Ipce docks

### PR DESCRIPTION
**Pull request by TracieSucharski**

### CODE REVIEW

File: isis/src/qisis/objs/Footprint2DView/Footprint2DView.h [hunk 1]
```diff
@@ -165,7 +171,7 @@ namespace Isis {
       QToolBar *m_activeToolBar; //!< The active tool bar
       ToolPad *m_toolPad; //!< The tool pad
 
-      QAction *m_controlNetTool;  //!< The Control Point Editor Tool
+      QAction *m_controlNetToolAction;  //!< The action for Control Point Editor Tool(MosaicControl
```
Comment (kaitlyndlee) [inline isis/src/qisis/objs/Footprint2DView/Footprint2DView.h]: I forgot to remove this with my last PR. This is not used anymore, so we can remove it.

**Review by kaitlyndlee — COMMENTED** [review]
Just one comment. Otherwise looks good!

File: isis/src/qisis/apps/ipce/IpceMainWindow.h [hunk 2]
```diff
@@ -174,6 +174,15 @@ namespace Isis {
...
+   *                           being misplaced when opening a project. Fixes #5175.
+   *   @history 2018-07-12 Tracie Sucharski - Renamed the signal Directory::viewClosed to
+   *                           Directory::closeView since Directory does not close the view but
+   *                           indicate that the view needs closing.  This signal is now used by
+   *                           more than the cnetEditorView, so updated documentation.  Did a little
+   *                           cleanup on the removeView  method by removing some code that
+   *                           automatically happens due to connection made on destroyed signal.
    *                           project was in fullscreen or not when saved. If not, we call
    *                           showNormal() to restore the poject's window size. This also fixes the
    *                           warning/history tabs being misplaced when opening a project.
```
Comment (SgStapleton) [inline isis/src/qisis/apps/ipce/IpceMainWindow.h]: Something got really wonky with these history comments...

File: isis/src/qisis/objs/Footprint2DView/Footprint2DView.h [hunk 3]
```diff
@@ -84,6 +84,9 @@ namespace Isis {
    *                           Removed methods that returned menu and toolbar actions.
    *                           Made it so that on default and if there is no active control net,
    *                           the Control Net Tool will be disabled.
+   *                           Added enableControlNetTool(bool) so when an active control net is set,
+   *                           the tool becomes enabled.
```
Comment (SgStapleton) [inline isis/src/qisis/objs/Footprint2DView/Footprint2DView.h]: History comment wonkiness again...

**Comment by SgStapleton** [comment]
We made a few small modifications in history comments and removed code that had been deleted in another PR that somehow still existed in here.

**Pull request merged**